Simplified/en.lproj/Localizable.strings

```diff
@@ -157,6 +161,9 @@
 "DecreaseFontSize" = "Decrease font size";
 "IncreaseFontSize" = "Increase font size";
 "BrightnessSlider" = "Brightness slider";
+"PublisherDefaultButtonTitle" = "Pub";
```

Comment:
"Pub" = "Editore"; (if not enough space, maybe use "Edit.")
"Publisher's Default" = "Impostazioni dell'editore";
"Show the publisher-specified fonts and layout choices in this ebook" = "Usa le impostazioni specificate dall'editore per il testo e la formattazione di questo ebook";